lib/views/multiline.coffee

```diff
@@ -0,0 +1,21 @@
+class MessageLine extends HTMLElement
```

Comment:
Hear me out: classes make sense where several semantically different elements can have this class (that's why it's a class). If a class makes sense only applied to one semantic element, it's redundant, and can be replaced with HTML5 semantic (user-defined) element.
I could `document.createElement 'message-line'` there, but that feels weird in OO code.

LineElement doesn't increase memory complexity, since most of its methods are shared from base instances, and the rest would be allocated for any element anyway. It also doesn't affect time complexity that much. Try as I might, I can't see what's wrong with my approach... :confused: